DataCollector/GitCollector.php

```diff
@@ -13,17 +14,28 @@ class GitCollector extends DataCollector
     /** @var string */
     private $rootDir;
 
+    /** @var string */
+    private $gitBinaryPath;
+
     /**
      * Constructor
      *
      * @param string $rootDir
+     * @param string $gitBinaryPath
      */
-    public function __construct($rootDir)
+    public function __construct($rootDir,$gitBinaryPath=null)
     {
         $this->rootDir = realpath($rootDir);
         if (false === $this->rootDir) {
             throw new \InvalidArgumentException(sprintf('"%s" is not a valid path.', $rootDir));
         }
+
+        if($gitBinaryPath){
```

Comment:
The conditional is needed to only trigger `file_exists` when a path has been submitted